website/docs/r/dnat.html.markdown

```diff
@@ -11,6 +11,10 @@ description: |-
 Provides a vCloud Director DNAT resource. This can be used to create, modify,
 and delete destination NATs to map an external IP/port to an internal IP/port.
 
+~> **Note:** Please use [`vcd_nsxv_dnat`](/docs/providers/vcd/r/nsxv_dnat.html) resource if you work
+with advanced edge gateways. This resource may corrupt UI edited NAT rules when used with advanced
+edge gateways.
```

Comment:
I just read this one more time. Please switch sentences places and maybe simplify the end. E.g.:

"This resource may corrupt UI edited NAT rules when used with advanced edge gateways. Please use [`vcd_nsxv_dnat`](/docs/providers/vcd/r/nsxv_dnat.html) in that case."

Reply:
Done in DNAT and SNAT